test/integration/docker_test.go

```diff
@@ -158,3 +164,114 @@ func TestForceSystemdEnv(t *testing.T) {
 		validateContainerdSystemd(ctx, t, profile)
 	}
 }
+
+// TestDockerEnvContainerd makes sure that minikube docker-env command works when the runtime is containerd
+func TestDockerEnvContainerd(t *testing.T) {
+	t.Log("running with", ContainerRuntime(), DockerDriver(), runtime.GOOS, runtime.GOARCH)
+	if ContainerRuntime() != constants.Containerd || !DockerDriver() || runtime.GOOS != "linux" {
+		t.Skip("skipping: TestDockerEnvContainerd can only be run with the containerd amd64 runtime on Docker driver")
```

Comment:
```suggestion
		t.Skip("skipping: TestDockerEnvContainerd can only be run with the containerd runtime on Docker driver")
```
We're not excluding arm64 in the if atatement, so this is incorrect